<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>M64PSettings</class>
 <widget class="QMainWindow" name="M64PSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>467</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Mupen64Plus Configuration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tab_widget">
      <widget class="QWidget" name="plugins_tab">
       <attribute name="title">
        <string>Plugins</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="folder_group_box">
          <property name="title">
           <string>Plugin Folder</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="open_plugin_folder_btn">
             <property name="text">
              <string>Open</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="folder_path_field"/>
           </item>
           <item>
            <widget class="QToolButton" name="set_plugin_folder_btn">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="refresh_plugins_btn">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="plugin_group_box">
          <property name="title">
           <string>Plugins</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="core_lbl">
             <property name="text">
              <string>Core</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="core_plugin_box">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="video_lbl">
             <property name="text">
              <string>Video</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="video_plugin_box">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="audio_lbl">
             <property name="text">
              <string>Audio</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="audio_plugin_box">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="input_lbl">
             <property name="text">
              <string>Input</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="input_plugin_box">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="rsp_lbl">
             <property name="text">
              <string>RSP</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="rsp_plugin_box">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="v_spacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>41</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="button_box">
      <property name="standardButtons">
       <set>QDialogButtonBox::Close</set>
      </property>
      <property name="centerButtons">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>596</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>M64PSettings</receiver>
   <slot>on_close_pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_plugin_folder_btn</sender>
   <signal>clicked()</signal>
   <receiver>M64PSettings</receiver>
   <slot>on_open_plugin_folder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_plugin_folder_btn</sender>
   <signal>clicked()</signal>
   <receiver>M64PSettings</receiver>
   <slot>on_set_plugin_folder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refresh_plugins_btn</sender>
   <signal>clicked()</signal>
   <receiver>M64PSettings</receiver>
   <slot>on_reset_plugin_folder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_close_pressed()</slot>
  <slot>on_open_plugin_folder()</slot>
  <slot>on_set_plugin_folder()</slot>
  <slot>on_reset_plugin_folder()</slot>
 </slots>
</ui>
